testbench/common.py

```diff
@@ -752,6 +783,14 @@ def handle_retry_test_instruction(database, request, method):
                     grpc_code=StatusCode.INTERNAL,  # not really used
                     context=None,
                 )
+    # TODO(cathyo@): introduce Retry Test API method "storage.objects.download" for large file downloads
```

Comment:
Please use a GitHub issue numbers to track TODO items, in case you win the lottery and or leave us for greener pa$ture$